src/families/ripple/bridge/js.ts

```diff
@@ -297,33 +249,36 @@ const txToOperation =
     }
 
     const op: Operation = {
-      id: `${account.id}-${id}-${type}`,
-      hash: id,
+      id: `${account.id}-${hash}-${type}`,
+      hash: hash,
```

Comment:
Have you tried this on an account with existing operations? Won't there be duplicates with different ids? :thinking:

Reply:
Nevermind I guess id and hash have the same value?

Reply:
yeah, they are the same in the previous implem I just reuse it